README.md

```diff
@@ -41,6 +41,25 @@ For an even easier usage, just `#import <KVOController/NSObject+FBKVOController.
 [self.KVOController observe:clock keyPath:@"date" options:NSKeyValueObservingOptionInitial|NSKeyValueObservingOptionNew action:@selector(updateClockWithDateChange:)];
 ```
 
+## Swift
+
+KVOController works fine with Swift but the properties that you need to observer must be marked as `dynamic` check the following example:
+```Swift
+class TasksListViewModel: NSObject, TasksListViewModelProtocol{
```

Comment:
Nit: `TasksListViewModelProtocol` sounds like an implementation detail.
It is indeed nice way to write these, but not sure if it's beneficial for this example.
Also, there is a missing space before the opening bracket. :)